src/pinns_pde_solve.jl

```diff
@@ -617,7 +617,7 @@ function get_numeric_derivative(parameterless_type_θ)
         end
 end
 
-Base.Broadcast.broadcasted(::typeof(get_numeric_derivative(AbstractVector)), phi,u,x,εs,order,θ) = get_numeric_derivative(AbstractVector)(phi,u,x,εs,order,θ)
+Base.Broadcast.broadcasted(::typeof(get_numeric_derivative(AbstractArray)), phi,u,x,εs,order,θ) = get_numeric_derivative(AbstractArray)(phi,u,x,εs,order,θ)
```

Comment:
why is this broadcast overload necessary?

Reply:
in order to overload broadcasting for functions of `u` and `derivative`  in the internal representation
derivative.(some) ==  derivative(some) 
```
 Expr[:((cord, var"##θ#1508", phi, derivative, u, p)->begin
          begin
              let (θ,) = (cord[[1], :],)
                  (+).(derivative.(phi, u, cord, Vector{Float32}[[0.0049215667]], 1, var"##θ#1508"), u.(cord, var"##θ#1508", phi)) .- (cos).(θ)
              end
          end
      end)])
```

Reply:
I would like to get rid of this, but for now, like this